tools/check_jsonnet

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -e
```

Comment:
somehow the code is duplicated in this flie, please take a look.

Reply:
updated.